dev-packages/e2e-tests/test-applications/astro-5/src/components/Welcome.astro

```diff
@@ -1,6 +1,4 @@
 ---
-import astroLogo from '../assets/astro.svg';
-import background from '../assets/background.svg';
```

Comment:
hmm looks like biome doesn't play well with astro? These imports are not unused

Reply:
unused rule can be disabled in override for urls ending with .astro